### PR TITLE
Reuse vectors

### PR DESCRIPTION
**Pull request by andylokandy**
- Reuse vectors

### CODE REVIEW

File: minitrace/src/local/local_collector.rs [hunk 1]
```diff
@@ -77,3 +76,14 @@ impl Drop for LocalCollector {
         }
     }
 }
+
+impl Clone for LocalSpans {
```
Comment (zhongzc) [inline minitrace/src/local/local_collector.rs:80]: Why need it?

Comment (andylokandy) [reply minitrace/src/local/local_collector.rs:80]: Because we removed the derive.

Comment (andylokandy) [reply minitrace/src/local/local_collector.rs:80]: `Reusable<T>` doesn't implement `Clone`

**Comment by zhongzc** [comment]
Please update bench result 
image

**Pull request closed without merging**